Rakefile

```diff
@@ -32,3 +31,12 @@ task :run => :compile do
   system(RbConfig.ruby, *%w(-I ./lib test.rb))
 end
 
+task :test_dap do
+  ENV['RUBY_DEBUG_DAP_TEST'] = '1'
```

Comment:
This line can be moved under `Rake::TestTask.new(:test_dap) do |t|`

Reply:
Did you check if what you said work?

If I move `ENV['RUBY_DEBUG_DAP_TEST']` to Rake::TestTask.new(:test_dap) do |t|, it's always true even I run other rake task, though.

Reply:
It can run `rake test_dap` but it looks like it'll also [leak the env var](http://dftrjy.blogspot.com/2019/01/rake-testtask-exports-environment.html).